src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -258,6 +258,10 @@ def _flexible_server_params(command_group):
                            help='Server major version.')
                 c.argument('zone', options_list=['--zone, -z'],
                            help='Availability zone into which to provision the resource.')
+                c.argument('iops', type=int, options_list=['--iops'],
```

Comment:
could you use CLIArgumentType to abstract the argument definition and reuse them?

Reply:
Do you mean like the way we do it for local-context variables ? Also, just trying to understand how does that help apart from not being repeated in two commands where they serve as parameters ? Because for none of the parameters (which are common across the services - PG, MySQL and MariaDB) we don't use CLI argument() except in local context varaibles.

Reply:
I mean for `--iops` definition.  You could abstract it as below and then reuse it:
https://github.com/Azure/azure-cli/blob/b4c666c9f9fc07eb98b85949e84184a3eddca9b0/src/azure-cli/azure/cli/command_modules/storage/_params.py#L86-L87

https://github.com/Azure/azure-cli/blob/b4c666c9f9fc07eb98b85949e84184a3eddca9b0/src/azure-cli/azure/cli/command_modules/storage/_params.py#L221